e2e-tests/kuttl/scheduled-backup/04-check-data.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
`04-check-data.yaml` should probably be `TestAssert` step with filename like `...-assert.yaml`. This way we would have all the checks in assert files and all test steps in other files.
That is if I understand correctly that this is already implemented: https://github.com/kudobuilder/kuttl/blob/d2cd0d5ec86a758ca4a13db032a89ae214d3d524/keps/0007-command-asserts.md

Reply:
Done. 0N-assert.yaml is a strict requirement. I was just wrong